terraform/modules/mongodb/helm/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "helm_release" "custom_helm_release" {
```

Comment:
Entire `mongodb/helm` folder can be deleted when using correct path format in `terraform block` as mentioned in other comment.

Reply:
done